### PR TITLE
feat: Add Mobitru cloud provider

### PR DESCRIPTION
**Pull request by vliubezny**
This PR adds support for [Mobitru](https//mobitru.com) cloud provider:

image

### CODE REVIEW

File: app/renderer/components/Session/ServerTabMobitru.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+const mobitruBillingUnitPlaceholder = (t) => {
+  if (process.env.MOBITRU_BILLING_UNIT) {
+    return t('usingDataFoundIn', {environmentVariable: 'MOBITRU_BILLING_UNIT'});
+  }
+  return 'personal';
+};
+
+const mobitruAccessKeyPlaceholder = (t) => {
+  if (process.env.MOBUTRU_ACCESS_KEY) {
```
Comment (eglitise) [inline app/renderer/components/Session/ServerTabMobitru.js:20]: I assume this typo is not intentional? :)

**Review by eglitise — APPROVED** [review]
Thank you!
Code looks good - could you please sign the CLA?

File: app/renderer/components/Session/ServerTabMobitru.js (hunk 1 above)
Comment (vliubezny) [reply app/renderer/components/Session/ServerTabMobitru.js]: Fixed. Thank you!

**Comment by mykola-mokhnach** [comment]
@vliubezny Could you please confirm the Mobitru company is not affiliated with Russian Federation or any of its beneficiary companies?

**Comment by vliubezny** [comment]
> @vliubezny Could you please confirm the Mobitru company is not affiliated with Russian Federation or any of its beneficiary companies?

[Mobitru](https://mobitru.com/) is owned by [EPAM Systems, Inc.](https://www.epam.com/). It's not affiliated with Russian Federation or any of its beneficiary companies.

I need to contact to responsible person in the company to sign the CLA. It might take some time.

**Comment by vliubezny** [comment]
@eglitise  The CLA is signed.

**Pull request merged**